src/Dofs/ConstraintHandler.jl

```diff
@@ -239,17 +236,35 @@ end
 
 Add the `AffineConstraint` to the `ConstraintHandler`.
 """
-function add!(ch::ConstraintHandler, newac::AffineConstraint)
-    # Basic error checking
-    for ac in ch.acs
-        (ac.constrained_dof == newac.constrained_dof) &&
-            error("Constraint already exist for dof $(ac.constrained_dof)")
-        any(x -> x.first == newac.constrained_dof, ac.entries) &&
-            error("New constrained dof $(newac.constrained_dof) is already used as a master dof.")
-    end
-    push!(ch.acs, newac)
-    push!(ch.prescribed_dofs, newac.constrained_dof)
-    push!(ch.inhomogeneities, newac.b)
+function add!(ch::ConstraintHandler, ac::AffineConstraint)
+    # TODO: Would be nice to pass nothing if ac.entries is empty, but then we lose the fact
+    #       that this constraint is an AffineConstraint which is currently needed in update!
+    #       in order to not update inhomogeneities for affine constraints
+    add_prescribed_dof!(ch, ac.constrained_dof, ac.b, #=isempty(ac.entries) ? nothing : =# ac.entries)
```

Comment:
If ac.entries is empty then it is a pure dirichlet, so note sure such a AffineConstraint should/would ever be added